src/Akeneo/Platform/CommunityVersion.php

```diff
@@ -12,7 +12,7 @@
 class CommunityVersion
 {
     /** @staticvar string */
-    const VERSION = '3.1.0';
+    const VERSION = '3.1.0-BETA1';
```

Comment:
```suggestion
    const VERSION = '3.2.0';
```
Because master is for 3.2